src/Collection/Collection.model.ts

```diff
@@ -122,28 +122,27 @@ export class Collection extends Model<CollectionAttributes> {
   static getPublishedJoinStatement(isPublished: boolean | undefined) {
     return isPublished !== undefined
       ? SQL`
-            JOIN ${raw(Item.tableName)} items ON items.collection_id = c.id ${
-          isPublished
-            ? SQL`AND (
-              (items.blockchain_item_id is NOT NULL AND c.third_party_id IS NULL)
-              OR c.third_party_id is NOT NULL)`
-            : SQL`AND (items.blockchain_item_id is NULL AND c.third_party_id IS NULL)`
-        }
-            LEFT JOIN ${raw(
-              ItemCuration.tableName
-            )} item_curations ON item_curations.item_id = items.id
-            ${
-              isPublished === false
-                ? SQL`
-                  WHERE (
-                    (SELECT COUNT(*) FROM item_curations
-                      LEFT JOIN items on items.id = item_curations.item_id
-                      LEFT JOIN collections cc on cc.id = items.collection_id
-                      WHERE items.collection_id = c.id AND item_curations.item_id = items.id
-                    ) = 0)
-                `
-                : SQL``
-            }
+          LEFT JOIN ${raw(Item.tableName)} items ON items.collection_id = c.id
```

Comment:
Modify the Join to a Left Join because the first one excluded the collections without items.